account_invoice_mgmt_dms/models/dms_file.py

```diff
@@ -39,8 +41,21 @@ def approve_account_move(self):
         self.account_move_id.message_post(
             body=_("Approve Invoice")
         )
+    
+    def validate_account_move(self):
+        for record in self:
+            if not self.env.user.has_group("account_invoice_mgmt_dms.group_invoice_approver") and self.env.user.has_group("account_invoice_mgmt_dms.group_invoice_validator") and record.account_move_id.validator_complete_proceesing_id != self.env.user:
+                raise ValidationError(_("You don't have permission to validate this invoice, only can validate the user: %s") % (record.account_move_id.validator_complete_proceesing_id.name))
```

Comment:
This method `validate_account_move` should be multi (it's right, because it could be called from more than one DMS file across a server action). Then, validation error message should be include invoice name, in order to better identification of raised error.